dumpgenerator.py

```diff
@@ -1641,7 +1648,19 @@ def saveSpecialVersion(config={}, session=None):
 
 def saveIndexPHP(config={}, session=None):
     """ Save index.php as .html, to preserve license details available at the botom of the page """
-
+    escaped_index = "'" + config['index'].replace("'", "'\\''") + "'"
+    escaped_path = "'" + (config['path'] + '/requisites').replace("'", "'\\''") + "'"
```

Comment:
Why not use subprocess.check_call? This seems simple enough a command, should work I think.